src/main/java/frc/robot/classes/spikemotor/SpikeMotorCANSparkMax.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.classes.spikemotor;
+
+import com.revrobotics.*;
+import frc.robot.classes.SPIKE293Utils;
+import static frc.robot.Constants.DrivetrainConstants.*;
+
+public class SpikeMotorCANSparkMax extends SpikeMotor {
+    private CANSparkMax motor;
+    private SparkMaxPIDController pidController;
+    private double wheelDiameter;
+    private boolean isBrushed;
+
+    public SpikeMotorCANSparkMax(double wheelDiameter, boolean isBrushed) {
+        this.wheelDiameter = wheelDiameter;
+        this.isBrushed = isBrushed;
+    }
+
+    @Override
+    protected void initImpl(int deviceNumber) {
```

Comment:
I would like to see some method description comments.

Reply:
Something like "Initializes this motor", "Sets the motor's speed", etc.?